src/uwtools/api/config.py

```diff
@@ -36,45 +42,55 @@ def compare(
 
 def get_fieldtable_config(config: Union[dict, OptionalPath] = None) -> _FieldTableConfig:
     """
-    Get a FieldTableConfig object.
+    Get a ``FieldTableConfig`` object.
 
-    :param config: Config file to load (None => read from stdin), or initial dict.
+    :param config: Config file to load (``None`` or unspecified => read ``stdin``), or initial
+        ``dict``.
```

Comment:
Just to make sure I understand the backtick problem you described above, this is an example of where you've manually wrapped a line containing backticks? Or did I get that wrong?

Reply:
Yes, I manually wrapped this line. But it's not just the lines with backticks that have to be manually wrapped: `docformatter` will not format a docstring _at all_ if it contains any backticks, anywhere in the docstring, as far as I can tell.

But since we only need backticks in docstrings in the API docs, docstrings everywhere except in `.py` files under `src/uwtools/api` are still fine: `docformatter` will continue to format them. And hopefully we'll get a bugfix from the `docformatter` maintainers.